expression/builtin_time_vec.go

```diff
@@ -2237,11 +2237,62 @@ func (b *builtinTimestampDiffSig) vecEvalInt(input *chunk.Chunk, result *chunk.C
 }
 
 func (b *builtinUnixTimestampIntSig) vectorized() bool {
-	return false
+	return true
 }
 
+// evalInt evals a UNIX_TIMESTAMP(time).
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_unix-timestamp
 func (b *builtinUnixTimestampIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETDatetime, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+
+	result.ResizeInt64(n, false)
+	i64s := result.Int64s()
+
+	if err := b.args[0].VecEvalTime(b.ctx, input, buf); err != nil {
+		var isNull bool
+		flag := terror.ErrorEqual(types.ErrWrongValue.GenWithStackByArgs(types.TimeStr, buf), err)
+		for i := 0; i < n; i++ {
+			i64s[i], isNull, err = b.evalInt(input.GetRow(i))
+			if err != nil {
+				return err
+			}
+			if flag {
+				// Return 0 for invalid date time.
+				i64s[i] = 0
+				continue
+			}
```

Comment:
Sorry it seems to be a mistake make by me.
We do not need judge here because `b.evalInt` already set it.

Reply:
em. thank you for your suggest. removed it